migration-util/src/main/java/edu/unc/lib/dcr/migration/premis/DepositRecordPremisToRdfTransformer.java

```diff
@@ -72,8 +72,8 @@ public void compute() {
                 addVirusCheckEvent(eventEl);
             } else if (eventTypes.contains(Premis2Constants.INGESTION_TYPE)) {
                 addIngestionEvent(eventEl);
-            } else if (eventTypes.contains(Premis2Constants.NORMALIZATION_TYPE)) {
-                transformNormalizationEvent(eventEl);
+            } else if (eventTypes.contains(Premis2Constants.ACCESSION_TYPE)) {
```

Comment:
This defines what we are migrating from, and we are migrating from boxc3 data which currently uses the Normalization event, so you'll need to leave it as NORMALIZATION in this case. We would only need to change the event type as it is stored in boxc5